stable/sonarqube/templates/copy-plugins.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
     heritage: {{ .Release.Service }}
 data:
   copy_plugins.sh: |-
-      #! /bin/sh
+      #!/bin/bash
```

Comment:
why?

Reply:
I linked the issue it resolves in the PR: https://github.com/helm/charts/issues/13482